core/services/src/main/java/org/exoplatform/services/cms/documents/impl/DocumentServiceImpl.java

```diff
@@ -899,7 +900,9 @@ public long getEditorsIdleTimeout() {
       if (userPrivateNode.hasNode(NodetypeConstant.FAVORITE)) {
         favoriteFolder = ((Node) userPrivateNode.getNode(NodetypeConstant.FAVORITE)).getPath();
       }
-      return getDocumentsByFolder(favoriteFolder, null, limit);
+      List<Document> favoriteDocuments = getDocumentsByFolder(favoriteFolder, null, limit);
```

Comment:
Why have we to filter on document in Security folder ?
Why documents in trash are not displayed ? How ? There is no filter to know if a document is in trash

Reply:
This test is used to check documents on the snapshot page favorite documents.

Reply:
You do not answer to my question :)
Why documents in trash are not displayed ? How ? There is no filter to know if a document is in trash
Why there is not filter to trash ? 

I want to understand how it work for trash to be sure to be consistent.

My idea is that for trash, favorites are removed, so, there is not need to add a filter. Can you confirm ?